aiven/client/cli.py

```diff
@@ -2675,6 +2675,51 @@ def credits__claim(self):
         if self.args.json:
             self.print_response(result, json=True)
 
+    @arg.json
+    @arg.project
+    @arg("--service", help="Related service name")
+    @arg("--severity", required=True, choices=["low", "high", "critical"], help="Ticket severity")
+    @arg("--title", required=True, help="Short description")
+    @arg("--description", required=True, help="Longer description")
+    def ticket_create(self):
+        """Create a support ticket"""
+        project_name = self.get_project()
+        ticket = self.client.create_ticket(
+            description=self.args.description,
+            project=project_name,
+            service=self.args.service,
+            severity=self.args.severity,
+            title=self.args.title,
+        )
+        self.print_response(result=ticket)
+
+    @arg.json
+    @arg.project
+    @arg("--state", required=False, help="Ticket state", choices=["closed", "open"])
+    def ticket_list(self):
+        """List support tickets for a project"""
+        project_name = self.get_project()
+        result = self.client.list_tickets(project=project_name)
+
+        tickets = result["tickets"]
+        if self.args.state:
+            tickets = [ticket for ticket in tickets if ticket["state"] == self.args.state]
+
+        layout = [
+            "ticket_id",
+            "severity",
+            "state",
+            "title",
+            "project_name",
+            "service_name",
+            "create_time",
+            "description",
+            "update_time",
+            "user_email",
+            "user_real_name"
+        ]
+        self.print_response(result=tickets, table_layout=layout, json=False)
```

Comment:
`json=self.args.json` here too if the command is to support raw json output with `--json`